config/images/package-loader.Containerfile

```diff
@@ -1,9 +1,9 @@
-FROM scratch
+FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:574f201d7ed185a9932c91cef5d397f5298dff9df08bc2ebb266c6d1e6284cd1
```

Comment:
Should this be done for all images in `config/`?

Reply:
ohh whoops! I missed to revert this.

A few days back I used the ubi-minimal image for some blocker but that blocker sorted out and I forgot to revert it back to scratch.

Will do that :)

Reply:
So, the package-loader image requires command like `mkdir` and `cp` to be executed against it which aren't supported under a scratch container. Therefore, I proceeded with the ubi-minimal container.